packages/perps-exes/src/bin/perps-bots/util/oracle.rs

```diff
@@ -316,9 +320,14 @@ fn compose_oracle_feeds(
                 sei.price.into_decimal256()
             }
             SpotPriceFeedData::Rujira { asset } => {
+                let asset = if network == &PerpsNetwork::RujiraDevnet {
```

Comment:
Oh, now I see the bot issue from last week. I'm wondering if I gave you a bad direction by saying we should parse out the asset name... it seems awkward to have a difference between the configured name and the name used in this map. It's still possible to keep the original name in the oracle price map though. Or perhaps it's fine to do it like this, and once we migrate contracts, we just update the spot price config with the shorter names.

Reply:
No, we shouldn't, because the motivation for using `BTC.BTC` is so that we don't need to update the spot price config at the same time that we migrate contracts. Keeping it as BTC.BTC gives us a test case for what the mainnet migration will look like.